README.md

```diff
@@ -3,6 +3,7 @@ cmd2: a tool for building interactive command line apps
 [![Latest Version](https://img.shields.io/pypi/v/cmd2.svg?style=flat-square&label=latest%20stable%20version)](https://pypi.python.org/pypi/cmd2/)
 [![Build status](https://img.shields.io/travis/python-cmd2/cmd2.svg?style=flat-square&label=unix%20build)](https://travis-ci.org/python-cmd2/cmd2)
 [![Appveyor build status](https://img.shields.io/appveyor/ci/FedericoCeratto/cmd2.svg?style=flat-square&label=windows%20build)](https://ci.appveyor.com/project/FedericoCeratto/cmd2)
+[![VSTS Build status](https://python-cmd2.visualstudio.com/cmd2/_apis/build/status/cmd2-Python%20package-CI?branch=master)](https://python-cmd2.visualstudio.com/cmd2/_build/latest?definitionId=1&branch=master)
```

Comment:
The cmd2 PR build/test CI system is now also integrated with Visual Studio Team Services (VSTS).  Currently it is doing builds for Python 3.6 and 3.7 on Linux VMs on a Windows host which is redundant with the TravisCI testing.  But the idea is that eventually we will use VSTS to do our macOS testing once Microsoft's support for that feature is more complete.